build.sbt

```diff
@@ -147,9 +149,8 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform).crossType(
   .jsSettings(commonJsSettings:_*)
   .jvmSettings(commonJvmSettings:_*)
   .jvmSettings(scoverageSettings:_*)
-  .jvmSettings(skip in publish := platform != JVMPlatform)
-  .jsSettings(skip in publish := platform != JSPlatform)
-  .nativeSettings(skip in publish := platform != NativePlatform)
+  .jvmSettings(skip in publish := isCustomScalaJSVersion)
```

Comment:
This is to prevent releasing JVM/Native artifacts a second time when the JS 0.6 release is happening.